src/puppetlabs/puppetdb/config.clj

```diff
@@ -316,13 +317,20 @@
   (let [default-web-router-service
         {:puppetlabs.puppetdb.cli.services/puppetdb-service "/pdb/query"
          :puppetlabs.puppetdb.dashboard/dashboard-service "/pdb"
-         :puppetlabs.puppetdb.metrics/metrics-service "/metrics"}]
+         :puppetlabs.puppetdb.metrics/metrics-service "/metrics"}
+        bootstrap-cfg (-> (find-bootstrap-config config-data)
+                          slurp
+                          str/split-lines)
+        dashboard-redirect? (contains? (set bootstrap-cfg)
```

Comment:
it's not better in theory. contains? is a log(n) set lookup and some is a linear vector scan. In practice I don't think it makes a difference here at all.